2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,64 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+const recipe1 = {
+  title: "Mole",
+  servings: 2,
+  ingredients: [
+    "cinnamon",
+    "cumin",
+    "cocoa"
+  ],
+};
+
+const recipe2 = {
+  title: "Tabbouleh",
+  servings: 2,
+  ingredients: [
+    "parsley",
+    "mint",
+    "Onions", 
+    "Tomatoes"
+  ],
+};
+
+const recipe3 = {
+  title: "Cheese Borek",
+  servings: 6,
+  ingredients: [
+    "500g Yufka sheets (or filo pastry)",
+    "250g Butter",
+    "Sunflower oil",
+    "1 Egg yolk",
+    "Sunflower oil",
+  ],
+};
+
+const recipe4 = {
+  title: "Lahmacun",
+  servings: 4,
+  ingredients: [
+    "200g Flour",
+    "2g Baker's yeast",
+    "15cl Lukewarm Water",
+    "Salt",
+  ],
+};
+
+const recipe5 = {
+  title: "Kalam Polo",
+  servings: 2,
+  ingredients: [
+    "rice", 
+    "dill", 
+    "Chives", 
+    "Vegetable oil"
+  ],
+};
+
```

Comment:
The recipe objects look good - but have another look at the instructions on lines 13-20. It is asking you to output each recipe like this:
```
  Mole
  Serves: 2
  Ingredients:
  cinnamon
  cumin
  cocoa
```
Can you think of how to do that without having too much duplicate code? 🤔